src/d3a/models/strategy/external_strategies/__init__.py

```diff
@@ -546,5 +547,6 @@ def market_info_dict(self) -> Dict:
                     if self.area.current_market else None,
                     "total_cost": self.energy_traded_costs(self.area.current_market.id)
                     if self.area.current_market else None},
-                "asset_bill": self.device.stats.aggregated_stats.get("bills")
+                "asset_bill": self.device.stats.aggregated_stats.get("bills"),
+
```

Comment:
Is it also due to pylint?

Reply:
Unfortunately ended up here during the implementation, Pylint does not complain about that. Should not hurt to have here though